haproxy.cfg

```diff
@@ -34,14 +34,12 @@ frontend router
   # "health" checks
   http-request return status 200 content-type text/plain string ok if { path /__nginxheartbeat__ } || { path /__lbheartbeat__ }
 
-  # Route to a backend based on path's prefix and content-type header
```

Comment:
nit: I think we should keep the explicit ACL as we are using the same pattern for other `hdr` checks above

Reply:
there are no other ACL...

Reply:
my bad, I was looking at a different version of the file on my local machine. You're absolutely correct.